src/extension-runners/chromium.js

```diff
@@ -151,12 +151,41 @@ export class ChromiumExtensionRunner {
     });
   }
 
-  wssBroadcast(data: Object) {
-    for (const client of this.wss.clients) {
-      if (client.readyState === WebSocket.OPEN) {
-        client.send(JSON.stringify(data));
+  async wssBroadcast(data: Object) {
+    return new Promise((resolve) => {
+      const clients = this.wss ? new Set(this.wss.clients) : new Set();
+
+      const webExtReloadComplete = async (message) => {
+        const msg = JSON.parse(message);
+
+        if (msg.type === 'webExtReloadExtensionComplete') {
+          for (const client of clients) {
+            client.off('message', webExtReloadComplete);
```

Comment:
I noticed that the tests fail on Node 8. That is because EventEmitter's `off` method was only introduced in Node 10.
We're going to drop support for Node 8 soon, because Node 8 is EOL in 2 months from now (https://github.com/nodejs/Release#release-schedule ).

If you want to land your patch *before* we drop Node 8 support, then you need to change `.off` to `removeListener` (3x)

Reply:
We have dropped for Node 8:  https://github.com/mozilla/web-ext/commit/884106c65f5e89a9e3c36b2dbb0609a1af27501d
... so you can use `.off` again.

Reply:
I'm getting some timeout errors when change to on/off. I think we could keep add/remove listener keywords.